app/services/throttler/is_throttled.rb

```diff
@@ -0,0 +1,32 @@
+module Throttler
+  class IsThrottled
+    def initialize(user_id, throttle_type)
```

Comment:
Looks like you could hardcode `throttle_type` in these service, right?

Reply:
Not sure what you mean.  You pass in the throttle type.

Reply:
Ah, nevermind, I see. I was misreading these.